src/hydra/fetchHydra.ts

```diff
@@ -0,0 +1,85 @@
+import { HttpError } from 'react-admin';
```

Comment:
You probably should `git mv` those files instead of deleting and recreating them. This would preserve git history.

Reply:
Done (i used 1 commit to revert back to js, then 1 commit to rename to ts/tsx using `git mv`)